hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystemStore.java

```diff
@@ -605,6 +614,29 @@ public void deleteFilesystem(TracingContext tracingContext)
     }
   }
 
+  /**
+   * Checks if we are creating a normal blob or markerFile.
+   * @param metadata takes metadata as param.
+   * @return true or false.
+   */
+  private boolean checkIsBlobOrMarker(HashMap<String, String> metadata) {
+    return metadata != null && TRUE.equalsIgnoreCase(metadata.get(X_MS_META_HDI_ISFOLDER));
+  }
+
+  private AbfsRestOperation createPath(final String path, final boolean isFile, final boolean overwrite,
+                                      final String permission, final String umask,
+                                      final boolean isAppendBlob, final String eTag,
+                                      TracingContext tracingContext) throws AzureBlobFileSystemException {
+    return client.createPath(path, isFile, overwrite, permission, umask, isAppendBlob, eTag, tracingContext);
```

Comment:
any reason for the method. lets remove it?

Reply:
Sneha had a comment to add abstraction for these repetitive methods.

Reply:
its one line command. How its helping?